### DISCUSSION TITLE
Plugins

### DISCUSSION THREAD

**Discussion by katastrophe92** [yoshiask/FluentStore#62] [Q&A]
Hello FluentStore community! This project is amazing and I that you could maybe work with the developer of WingetUI. This 2 projects together=the only windows app store everybody needs. Just 1 question: what other plug-ins are besides the preinstalled ones? Thanks

**Comment by yoshiask** [comment]
So far, there are no third-party plugins, just the ones in this repo. I'd be happy to assist others with making plugins! Also, now that 0.4.0-beta is out, I am planning to investigate adding more points of extensibility so plugins can do more than just provide access to a storefront.
